_data/apps.yml

```diff
@@ -820,3 +820,13 @@
     - "email"
     - "productivity"
   icon: "tiliq.png"
+-
+  name: "PhotoScreenSaver"
+  description: "Windows screen saver that displays a photo slideshow"
+  website: "https://github.com/RandScullard/photo-screen-saver"
+  repository: ""
+  keywords:
+    - "screensaver"
+    - "photography"
+  license: "MIT"
+  icon: "photo-screen-saver.png"
```

Comment:
Sorry, this should be mentioned in the docs, but can you update this image to be `256x256px`?